test/parallel/test-http-client-close-with-default-agent.js

```diff
@@ -20,4 +20,4 @@ server.listen(0, common.mustCall(() => {
 }));
 
 // This timer should never go off as the server will close the socket
-setTimeout(common.mustNotCall(), common.platformTimeout(10000)).unref();
+setTimeout(common.mustNotCall(), common.platformTimeout(1000)).unref();
```

Comment:
Does this change help at all? Without looking much into what the test is doing, it looks like this timeout should not keep the process alive.

Reply:
> it looks like this timeout should not keep the process alive.

Correct but the test should finish after 1 sec. If it does not finish after 1 sec, then there is something wrong. The delay was taken from the removed test. We can use a larger delay but 10 sec is too much in my opinion.